.github/workflows/build.yml

```diff
@@ -291,24 +291,30 @@ jobs:
           cd build
           ctest -C Release --verbose --timeout 900
 
-      - name: Get commit hash
-        id: commit
-        if: ${{ ( github.event_name == 'push' && github.ref == 'refs/heads/master' ) || github.event.inputs.create_release == 'true' }}
-        uses: pr-mpt/actions-commit-hash@v2
+      - name: Determine tag name
+        id: tag
+        shell: bash
+        run: |
+          BUILD_NUMBER="$(git rev-list --count HEAD)"
+          if [[ "${{ env.BRANCH_NAME }}" == "master" ]]; then
+            echo "name=$BUILD_NUMBER" >> $GITHUB_OUTPUT
+          else
+            echo "name=${{ env.BRANCH_NAME }}-$BUILD_NUMBER" >> $$ GITHUB_OUTPUT
```

Comment:
`$$ GITHUB_OUTPUT` looks wrong :)

Reply:
Good catch. How did it even work like that? That's the exact build.yml that succeeded in the forked repo.